x/examples/test-connectivity/main.go

```diff
@@ -80,7 +92,8 @@ type errorJSON struct {
 	// Posix error, when available
 	PosixError string `json:"posix_error,omitempty"`
 	// TODO: remove IP addresses
-	Msg string `json:"msg,omitempty"`
+	Msg     string `json:"msg,omitempty"`
```

Comment:
We need to note that these two fields may contain Personally Identifiable Information (PII). We need to figure out a way to not leak that before this is actually used in production with user probes.

We should probably allowlist specific errors as we learn about them. Though we need the full message to learn about them.